kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/collections/ShouldContainOnlyTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.sksamuel.kotest.matchers.collections
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.shouldContainOnly
+import io.kotest.matchers.collections.shouldNotContainOnly
+import io.kotest.matchers.throwable.shouldHaveMessage
+
+class ShouldContainOnlyTest : WordSpec() {
+
+    init {
+        "containOnly" should {
+            "test that an array contains given elements only" {
```

Comment:
Would it be possible to break this up into smaller tests, rather than one big test?

Reply:
sure, I will make the changes

Reply:
sure, I will make the changes